testing/kubectl_helper.go

```diff
@@ -59,6 +63,16 @@ func (k *Kubectl) Create(namespace string, yamlFilePath string) error {
 	return nil
 }
 
+// CreateSecretFromLiteral creates a generic type secret using kubectl command
+func (k *Kubectl) CreateSecretFromLiteral(namespace string, secretName string, secretType string, literalValues string) error {
```

Comment:
I think this only works for `generic` secrets anyway, no? So this parameter could be omitted.

`literalValues` implies that it's possible to set multiple values while it's only possible to set one key-value pair afaik. I'd change it to singular.

I think it's also better to just take the payload as an parameter here and not expect the caller to add the `--from-literal=` part.

Reply:
Yup. Gotcha. Done